<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Search Parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="authorsCheck">
          <property name="text">
           <string>By Author(s):</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="authorsEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="yearCheck">
          <property name="text">
           <string>Year Range:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QSpinBox" name="yearStartSpin">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1960</number>
            </property>
            <property name="maximum">
             <number>2040</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>to</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="yearEndSpin">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1960</number>
            </property>
            <property name="maximum">
             <number>2040</number>
            </property>
            <property name="value">
             <number>2040</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="keywordsCheck">
          <property name="text">
           <string>Keywords:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="keywordsReviewCheck">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Review Text</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="keywordsTitleCheck">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Title</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLineEdit" name="keywordsEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="paperPathCheck">
          <property name="toolTip">
           <string>Search for a paper</string>
          </property>
          <property name="text">
           <string>Paper on Disk:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLineEdit" name="paperPathEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Full path to paper file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="paperSelectButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Select the paper</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="BusyIndicator" name="busyWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Results</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="resultsStatusLabel">
          <property name="text">
           <string>No results found.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="citationResultsList"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BusyIndicator</class>
   <extends>QWidget</extends>
   <header>busyindicator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>yearStartSpin</tabstop>
  <tabstop>yearEndSpin</tabstop>
  <tabstop>keywordsReviewCheck</tabstop>
  <tabstop>keywordsTitleCheck</tabstop>
  <tabstop>keywordsEdit</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>citationResultsList</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>paperPathCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>paperPathEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>paperPathCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>paperSelectButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>authorsCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>authorsEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yearCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>yearStartSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yearCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>yearEndSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>keywordsCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>keywordsReviewCheck</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>keywordsCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>keywordsEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>keywordsCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>keywordsTitleCheck</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
